src/perf/lib/quicmain.cpp

```diff
@@ -0,0 +1,95 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    QUIC Perf Main execution engine.
+
+--*/
+
+
+
+#ifndef _KERNEL_MODE
+#define QUIC_TEST_APIS 1
+#endif
+#include "quic_driver_main.h"
+
+#include "ThroughputServer.h"
+#include "ThroughputClient.h"
+
+#include "quic_trace.h"
+#ifdef QUIC_CLOG
+#include "quicmain.cpp.clog.h"
+#endif
+
+const QuicApiTable* MsQuic {nullptr};
+
+PerfBase* TestToRun {nullptr};
```

Comment:
All global variable auto-initialize to zero'd out. No need to explicitly set nullptr.